### PR TITLE
Google auth

### PR DESCRIPTION
**Pull request by hecerinc**
- Adds google authentication via `itesm.mx`
- Validates the token on the server 
- Returns name temporarily and shows the user a small login message.

### CODE REVIEW

File: resources/views/login.blade.php [hunk 1]
```diff
@@ -3,14 +3,105 @@
...
+
+		function signOut() {
+			var auth2 = gapi.auth2.getAuthInstance();
+			auth2.signOut().then(function () {
+				console.log('User signed out.');
+				window.location.href="/";
+			});
+		}
+	</script>
+@endsection
```
Comment (katiearriagam) [inline resources/views/login.blade.php]: nit: missing final line

File: resources/views/layouts/app.blade.php [hunk 2]
```diff
@@ -25,7 +25,7 @@
                 <a class="item" href="/projects">
                     Projects
                 </a>
-                <a class="ui item" href="/login">
+                <a class="ui item" id="logoutBtn" href="/login">
```
Comment (katiearriagam) [inline resources/views/layouts/app.blade.php]: why is this if "logoutBtn" if it's login?

File: resources/views/login.blade.php (hunk 1 above)
Comment (hecerinc) [reply resources/views/login.blade.php]: Sí está ahí, en Github nos ale porque por default no se parsea.

File: resources/views/auth.blade.php [hunk 3]
```diff
@@ -11,19 +11,78 @@
...
+			<p>Your token goes here</p>
+			<input type="hidden" name="id_token" id="tokenInput">
+			<button type="submit">Submit</button>
+		</form>
 	</div>
+	<script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
 	<script src="https://apis.google.com/js/platform.js" async defer></script>
 	<script>
+		$(function() {
+
```
Comment (katiearriagam) [inline resources/views/auth.blade.php]: nit: blank line? maybe keep this consistent?

**Pull request merged**